### ISSUE TITLE
Site Session Location

### ISSUE THREAD

**Issue by charlie-prezzano** [hawthorne-git/hawthorne-admin#18]
The IP address used to get location (city, state, country) is off ... I think the root issue is that we are getting the ip address from cloudlfare, not the client.

**Comment by peter-hawthorne** [comment]
Yes, just found -> https://support.cloudflare.com/hc/en-us/articles/200170986-How-does-CloudFlare-handle-HTTP-Request-headers- and https://devcenter.heroku.com/articles/http-routing#heroku-headers to solve IP restoration.
